mcp_server/main.py

```
@@ -466,8 +467,12 @@
 
             if writer:
                 writer.write(response_str.encode())
-                await writer.drain()
-            else: # Fallback for Windows
+                try:
+                    await writer.drain()
+                except AttributeError:
+                    # Fallback for protocol without drain helper
+                    pass
```

Ruff (0.11.9)

Use `contextlib.suppress(AttributeError)` instead of `try`-`except`-`pass`

(SIM105)